site/src/api/typesGenerated.ts

```diff
@@ -175,6 +182,12 @@ export interface CreateWorkspaceRequest {
   readonly parameter_values?: CreateParameterRequest[]
 }
 
+// From codersdk/metrics.go
+export interface DAUEntry {
+  readonly date: string
+  readonly daus: number
```

Comment:
nit: `DAUEntry.daus` feels a bit weird. Would also prefer for us to use consistent casing if possible: `DAU` vs. `dau`

Maybe `total`?

Reply:
The casing is an unfortunate side effect of the code generator, which is outside the scope of this PR. I can rename to `total` if you feel strongly, but I think `daus` is clear enough.